bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafPanelHandler.java

```diff
@@ -91,7 +95,7 @@ public void initialize() {
 
     @Override
     public void bridgeStatusChanged(ThingStatusInfo controllerStatusInfo) {
-        logger.debug("Controller status changed to {}", controllerStatusInfo);
+        logger.debug("Controller status changed to {} -- {}", controllerStatusInfo, controllerStatusInfo.getDescription()+"/"+controllerStatusInfo.getStatus()+"/"+controllerStatusInfo.hashCode());

```

Comment:
Please pass them separate instead of joining them in the arguments, just add {}/{}/{}